app/javascript/legacy/nonprofits/supporters/index/tags_and_fields_shared_methods.js

```diff
@@ -5,14 +5,24 @@ var tags_or_fields = {}
 
 
 tags_or_fields.definition_endpoint = function (type) {
-	return endpoint_prefix + type + '_index_definitions'
+	if (type === 'tag') { 
+		return endpoint_prefix + type + '_masters';
+	}
+	else {
+		return endpoint_prefix + type + '_definitions';
+	}
```

Comment:
This code assumed that the end points for tag definitions and custom field definitions were identical, i.e. they would end with `tag_definitions` and `custom_field_definitions`.

The problem though is that with this change, they're not identical: tags use `tag_master` and custom fields use `custom_field_definition`. This change corrects the bugs but it can be removed once TagMaster gets renamed too.